### DISCUSSION TITLE
Can't see profile of merger or copy their name

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44036] [Mobile] [answered]
Browse to https://github.com/actions/checkout/pull/1057 in GitHub Mobile beta
![Screenshot_20230111-191219.png](https://user-images.githubusercontent.com/2119212/211945255-f6e9cf31-0ef7-4a95-ada9-f05525477352.png)

Try to copy the name of the merger so that they can be tagged to ask about https://github.com/actions/checkout/pull/465 which has been sitting lonely for over a year and a half.

### Actual results
I think the time of the event is copied or something similarly unhelpful.

### Expected results
Tapping a user should browse to that user's profile. 

If it's really important to expose timestamp and event links, then tapping an event should bring up a sheet from the bottom exposing the various useful details:
* Who (link, copyable)
* When (link, copyable)
* What (maybe a way to see the details of a what, e.g. a link to a commit)

**Comment by alcere** [comment, marked as answer]
Hi @jsoref, thanks for reporting! Fair point, I have shared this issue with the team, and they will add it to the backlog.

**Comment by ghost** [comment]
> Browse to https://github.com/actions/checkout/pull/1057 in GitHub Mobile beta
> ![Screenshot_20230111-191219.png](https://user-images.githubusercontent.com/2119212/211945255-f6e9cf31-0ef7-4a95-ada9-f05525477352.png)
> 
> Try to copy the name of the merger so that they can be tagged to ask about https://github.com/actions/checkout/pull/465 which has been sitting lonely for over a year and a half.
> 
> ### Actual results
> I think the time of the event is copied or something similarly unhelpful.
> 
> ### Expected results
> Tapping a user should browse to that user's profile. 
> 
> If it's really important to expose timestamp and event links, then tapping an event should bring up a sheet from the bottom exposing the various useful details:
> * Who (link, copyable)
> * When (link, copyable)
> * What (maybe a way to see the details of a what, e.g. a link to a commit)
>
